### DISCUSSION TITLE
include path needed for #include <cs50.h>

### DISCUSSION THREAD

**Discussion by jcontorno** [microsoft/vcpkg#26874] [Q&A]
I am having a terrible time with VScode for CS50.  I keep getting make not found and now include path needed for the two .h sets

Can someone please tell me what I should do to clean this mess up?

**Comment by StarGate-One** [comment]
Hi @jcontorno 

You might want to take a look at [How to use cs50.h in my local VSCode environment? #189](https://github.com/cs50/libcs50/issues/189) - it may shed some light on how to use vscode, gcc and cs50. Suggest reading through this a few times, especially this response by [timeneverdie](https://github.com/timeneverdie) commented [on Dec 2, 2020](https://github.com/cs50/libcs50/issues/189#issuecomment-737173845).
